elasticsearch/README.md

```diff
@@ -0,0 +1,51 @@
+# Elasticsearch
+
+## Image Utilities
+Following are utilities included in the image to facilitate running commonly used
+administration commands.  Use of these utilities assumes you have `exec` access
+to the container.  Some of these utilities make use of the administration certs which have
+full access to all indices and operations of the Elasticsearch cluster.
+
+### es_util
+Run any REST command against the Elasticsearch endpoint
+
+Example to retrieve the list of nodes
+```
+$ oc exec -c elasticsearch $POD -- es_util --query=_cat/nodes?pretty
```

Comment:
Had we considered just making the command an argument like:

```
$ oc exec -c elasticsearch $POD -- es_util _cat/nodes?pretty
```

Reply:
The command is formed this way because it uses query for obvious reasons and passes any additional args to curl as is so you can provide additional options.  It has been like that for a while and my bashfu is not strong enough to know how it could have been better implemented